src/Orleans.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
```

Comment:
I have only VS2017 now. Also that has no effect on the generated files, it is just the VS Version. Honestly, it shouldn't be there... Everyone that has an update of VS and open the solution will have this value changed. Hard to keep older ones...